dev/source/docs/ardupilot-on-linux-starting.rst

```diff
@@ -0,0 +1,58 @@
+.. _ardupilot-on-linux-starting:
+
+===============
+Startup Options
+===============
+
+Run a build with -\-help to see the latest command-line options.
+
+-\-serial#
+==========
+
+This command line argument, with # replaced by a number, is used to connect the an ArduPilot serial port to a serial device or network protocol.
+May also use old-style(-A, -B, ...).
+
+For each ArduPilot serial port, specify either a device file or a short string to configured the network protocol.
+
+Serial Ports: /dev/deviceFile
+	- -\-serial0 /dev/ttyAMA0
```

Comment:
We would need a new section to show how to get the peripheral by id and udev . Both are simple and solve the main issue people have with restarting and port changing location ! I can help doing the udev